### PR TITLE
pound 4.12 (new formula)

### PR DESCRIPTION
**Pull request by developedsoftware**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [C] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by developedsoftware** [comment]
Unsure how to pass the CI checks for this?

    /lib/x86_64-linux-gnu/libcrypt.so.1

File: Formula/p/pound.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  url "https://github.com/graygnuorg/pound/releases/download/v4.12/pound-4.12.tar.gz"
+  sha256 "3276463b3faacfa5c7f16daac2c3d7f9d5946b4505c12923c32e017b7229a1f2"
+  license "GPL-3.0-or-later"
+
+  depends_on "gperftools"
+  depends_on "openssl@3"
+  depends_on "pcre"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
```
Comment (SMillerDev) [inline Formula/p/pound.rb:13]:
```suggestion
    system "./configure", *std_configure_args
```

File: Formula/p/pound.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  desc "Reverse proxy, load balancer and HTTPS front-end for web servers"
+  homepage "https://github.com/graygnuorg/pound"
+  url "https://github.com/graygnuorg/pound/releases/download/v4.12/pound-4.12.tar.gz"
+  sha256 "3276463b3faacfa5c7f16daac2c3d7f9d5946b4505c12923c32e017b7229a1f2"
+  license "GPL-3.0-or-later"
+
+  depends_on "gperftools"
+  depends_on "openssl@3"
+  depends_on "pcre"
+
```
Comment (SMillerDev) [inline Formula/p/pound.rb:11]:
```suggestion
  depends_on "pcre"

  uses_from_macos "libxcrypt"

```

**Comment by developedsoftware** [comment]
Thanks @SMillerDev I think thats cracked it. 

I guess its just a waiting game now? `* GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)`

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**